packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -29,6 +29,45 @@ book.addMethod('GET');
 book.addMethod('DELETE');
 ```
 
+### AWS Lambda-backed APIs
+
+A very common practice is to use Amazon API Gateway with AWS Lambda as the
+backend integration. The `LambdaRestApi` construct makes it easy:
+
+The following code defines a REST API that uses a greedy `{proxy+}` resource
+mounted under `/api/v1` and integrates all methods (`"ANY"`) with the specified
+AWS Lambda function:
+
+```ts
+const backend = new lambda.Function(...);
+new apigateway.LambdaRestApi(this, 'myapi', {
+  handler: backend,
+  proxyPath: '/api/v1'
```

Comment:
That's not quite a `proxyPath` to me (because `/api/v1/{proxy+}` would be). This is more like a `basePath` or something like that.